app/posts/breast-screening-reporting-team/2025/02/2025-02-01-data-product-design-principles.md

```diff
@@ -0,0 +1,43 @@
+---
+  title: Data Product Design Principles
+  description: Ensuring data is usable and useful
+  date: 2025-02-01
+  tags:
+    - data
+    - reporting
+    - user needs
+---
+
+
+**Note: this work was completed by Team Analyse. It will be carried forward by Breast Screening Reporting.**
+
+There can be a perception that data is somehow independent of its users and if they just see some numbers that’s enough. Our research into data needs showed us that not all users can use what they are given, and therefore that rolling out a product or platform is not the same as adoption among users.
+
+## What we found
+
+In Breast Screening Offices (BSOs) we saw that some data was:
+
+- difficult to access due to problematic logins, for example
+- not specific enough to them to be useful
+- difficult to use due to the complexity of the tool
+- communicated using language and concepts users didn’t always understand
+- did not contain all the data needed to answer their questions
+
+In a busy work context these can act as big barriers to valuable data.
+
+In addition to data in tools and platforms there are issues with accessing data in their own system, the National Breast Screening System (NBSS) such as:
+
+- specialised skills are needed to get data out using a second system
+- often when data is extracted, BSO admin staff users spend time manually ‘wrangling’ data before it is usable
+
+## What we did
+
+This evidence suggested we need some design principles to ‘keep us honest’ into the future and ensure that we are always meeting the needs of users, not just supplying the right numbers. Users must be able to easily access, understand and act on the data they need.
+
+The draft of design principles below was inspired by NHS design principles and adapted for data users in breast screening.
+
+![Data product design principles - version 1](data-product-design-principles.png)
```

Comment:
An image of text isn't ideal for accessibility. Any other options? otherwise, could the text of the principles be included as text somewhere - alt text, table, etc.